plugins/@grouparoo/postgres/__tests__/integration/log-checking.ts

```diff
@@ -39,6 +39,7 @@ async function getPropertyValue(query: string) {
 
 describe("postgres/integration/log-checking", () => {
   helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
   const actionhero = require("actionhero");
   const logMock = jest.fn();
   Object.defineProperty(actionhero, "log", { value: logMock, writable: false });
```

Comment:
Ohh.... we are hacking the logger here :/

Reply:
This was some of my first work here. If there was a better way to do it, I was unaware at the time.

Reply:
Something similar had to be done in core for testing export logs 

https://github.com/grouparoo/grouparoo/blob/a9ce0f6c516dffa8e6d54659eb153696f6a91039/core/__tests__/models/export.ts#L576-L585